tests/pipeline-all-amps/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+IMAGE_TAG=7.0.0-M2
```

Comment:
Can we get this from a pom file somewhere?  Is there a reason why we're using 7.0.0-M2 rather than 7.0.0-A14?

Reply:
I was just testing the replacement in local dev, not used by pipeline but ideally this should be used rather than the maven plugin that doesn't work with latest docker desktop on my mac